2018-05-16T04:01:07.916Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID '004' in collection with ID 'Asset:org.acme.Z2BTestNetwork.Order' does not exist)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-16T04:01:07.917Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID '004' in collection with ID 'Asset:org.acme.Z2BTestNetwork.Order' does not exist)","stack":"Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID '004' in collection with ID 'Asset:org.acme.Z2BTestNetwork.Order' does not exist)\n    at channel.queryByChaincode.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:809:34)\n    at <anonymous>"}$
2018-05-16T04:01:08.445Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID '007' in collection with ID 'Asset:org.acme.Z2BTestNetwork.Order' does not exist)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-16T04:01:08.445Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID '007' in collection with ID 'Asset:org.acme.Z2BTestNetwork.Order' does not exist)","stack":"Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID '007' in collection with ID 'Asset:org.acme.Z2BTestNetwork.Order' does not exist)\n    at channel.queryByChaincode.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:809:34)\n    at <anonymous>"}$
2018-05-16T04:01:09.051Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID '006' in collection with ID 'Asset:org.acme.Z2BTestNetwork.Order' does not exist)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-16T04:01:09.051Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID '006' in collection with ID 'Asset:org.acme.Z2BTestNetwork.Order' does not exist)","stack":"Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID '006' in collection with ID 'Asset:org.acme.Z2BTestNetwork.Order' does not exist)\n    at channel.queryByChaincode.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:809:34)\n    at <anonymous>"}$
2018-05-16T04:01:11.211Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID '001' in collection with ID 'Asset:org.acme.Z2BTestNetwork.Order' does not exist)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-16T04:01:11.211Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID '001' in collection with ID 'Asset:org.acme.Z2BTestNetwork.Order' does not exist)","stack":"Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID '001' in collection with ID 'Asset:org.acme.Z2BTestNetwork.Order' does not exist)\n    at channel.queryByChaincode.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:809:34)\n    at <anonymous>"}$
2018-05-16T04:01:11.424Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID '002' in collection with ID 'Asset:org.acme.Z2BTestNetwork.Order' does not exist)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-16T04:01:11.436Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID '002' in collection with ID 'Asset:org.acme.Z2BTestNetwork.Order' does not exist)","stack":"Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID '002' in collection with ID 'Asset:org.acme.Z2BTestNetwork.Order' does not exist)\n    at channel.queryByChaincode.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:809:34)\n    at <anonymous>"}$
2018-05-16T04:01:12.737Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Object with ID '008' in collection with ID 'Asset:org.acme.Z2BTestNetwork.Order' does not exist)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-16T04:01:12.737Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID '008' in collection with ID 'Asset:org.acme.Z2BTestNetwork.Order' does not exist)","stack":"Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: Object with ID '008' in collection with ID 'Asset:org.acme.Z2BTestNetwork.Order' does not exist)\n    at channel.queryByChaincode.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:809:34)\n    at <anonymous>"}$
2018-05-16T04:01:13.306Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'mary@iseriesexpertsinc.com' as the object already exists)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-16T04:01:13.307Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'mary@iseriesexpertsinc.com' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'mary@iseriesexpertsinc.com' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-16T04:01:13.402Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'abdul@cognitiveadvantageinc.com' as the object already exists)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-16T04:01:13.402Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'abdul@cognitiveadvantageinc.com' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'abdul@cognitiveadvantageinc.com' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-16T04:01:13.431Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'wilber@coolingsystemsrusinc.com' as the object already exists)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-16T04:01:13.432Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'wilber@coolingsystemsrusinc.com' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'wilber@coolingsystemsrusinc.com' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-16T04:01:13.443Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'bob@pchardwareinc.com' as the object already exists)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-16T04:01:13.443Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'bob@pchardwareinc.com' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'bob@pchardwareinc.com' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-16T04:01:13.547Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'jojo@innovativesolutionsinc.com' as the object already exists)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-16T04:01:13.547Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'jojo@innovativesolutionsinc.com' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'jojo@innovativesolutionsinc.com' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-16T04:01:13.624Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'info@pchardwarenowinc.com' as the object already exists)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-16T04:01:13.624Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'info@pchardwarenowinc.com' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'info@pchardwarenowinc.com' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-16T04:01:13.649Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'easymoney@easymoneyinc.com' as the object already exists)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-16T04:01:13.649Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'easymoney@easymoneyinc.com' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'easymoney@easymoneyinc.com' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-16T04:01:14.157Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'joshua@officeexpertsinc.com' as the object already exists)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-16T04:01:14.157Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'joshua@officeexpertsinc.com' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'joshua@officeexpertsinc.com' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-16T04:01:14.175Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'yes@softwaresolutionsinc.com' as the object already exists)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-16T04:01:14.175Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'yes@softwaresolutionsinc.com' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'yes@softwaresolutionsinc.com' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-16T04:01:14.391Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'aiesha@cloudninesoftwareinc.com' as the object already exists)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-16T04:01:14.391Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'aiesha@cloudninesoftwareinc.com' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'aiesha@cloudninesoftwareinc.com' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-16T04:01:14.407Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'Erin@2ndlifesystemsinc.com' as the object already exists)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-16T04:01:14.407Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'Erin@2ndlifesystemsinc.com' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'Erin@2ndlifesystemsinc.com' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-16T04:01:14.428Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'anytime@virtualpaperinc.com' as the object already exists)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-16T04:01:14.428Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'anytime@virtualpaperinc.com' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'anytime@virtualpaperinc.com' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-16T04:01:14.456Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'buyer1@hybridclouddesignsinc.com' as the object already exists)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-16T04:01:14.457Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'buyer1@hybridclouddesignsinc.com' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'buyer1@hybridclouddesignsinc.com' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-16T04:01:14.906Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'power@upssystemsinc.com' as the object already exists)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-16T04:01:14.907Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'power@upssystemsinc.com' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'power@upssystemsinc.com' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-16T04:01:15.047Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'noop@dummy' as the object already exists)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-16T04:01:15.047Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'noop@dummy' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'noop@dummy' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-16T04:01:15.060Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'alwayson@nonstopinc.com' as the object already exists)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-16T04:01:15.060Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'alwayson@nonstopinc.com' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'alwayson@nonstopinc.com' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-16T04:01:15.103Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'wantitnow@overnightexpressinc.com' as the object already exists)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-16T04:01:15.103Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'wantitnow@overnightexpressinc.com' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'wantitnow@overnightexpressinc.com' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-16T04:01:15.785Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'kristen@gameify.com' as the object already exists)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-16T04:01:15.785Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'kristen@gameify.com' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'kristen@gameify.com' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-16T04:01:15.847Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'eric@bornonthecloudinc.com' as the object already exists)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-16T04:01:15.847Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'eric@bornonthecloudinc.com' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'eric@bornonthecloudinc.com' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-16T04:01:15.872Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'noop@dummy' as the object already exists)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-16T04:01:15.872Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'noop@dummy' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'noop@dummy' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-16T04:01:16.675Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID '009' as the object already exists)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-16T04:01:16.675Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID '009' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID '009' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-16T04:01:16.688Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'speedmatters@fasteddyinc.com' as the object already exists)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-16T04:01:16.689Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'speedmatters@fasteddyinc.com' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'speedmatters@fasteddyinc.com' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-16T04:01:16.920Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'abby@kidfriendlyinc.com' as the object already exists)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-16T04:01:16.920Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'abby@kidfriendlyinc.com' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'abby@kidfriendlyinc.com' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-16T04:01:17.040Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID '005' as the object already exists)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-16T04:01:17.040Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID '005' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID '005' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-16T04:01:17.320Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'online@theappstoreinc.com' as the object already exists)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-16T04:01:17.320Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'online@theappstoreinc.com' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID 'online@theappstoreinc.com' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-16T04:01:17.511Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: Failed to add object with ID '003' as the object already exists)
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-16T04:01:17.511Z ERROR   HLFConnection           :invokeChainCode()        {"message":"Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID '003' as the object already exists)\n","stack":"Error: Error trying invoke business network. Error: No valid responses from any peers.\nResponse from attempted peer comms was an error: Error: chaincode error (status: 500, message: Error: Failed to add object with ID '003' as the object already exists)\n\n    at _initializeChannel.then.then.then.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:916:34)\n    at <anonymous>"}$
2018-05-16T04:01:32.191Z ERROR   FileSystemCardStore     :get()                    {"message":"Unable to read card directory: /home/damsen2/.composer/cards/abby@kidfriendlyinc.com","stack":"Error: Unable to read card directory: /home/damsen2/.composer/cards/abby@kidfriendlyinc.com\n    at newErrorWithCause (/home/damsen2/Chapter10/node_modules/composer-common/lib/idcard.js:36:19)\n    at fs.stat.catch.cause (/home/damsen2/Chapter10/node_modules/composer-common/lib/idcard.js:323:19)\n    at <anonymous>","cause":{"errno":-2,"code":"ENOENT","syscall":"stat","path":"/home/damsen2/.composer/cards/abby@kidfriendlyinc.com"}}$
2018-05-16T04:02:14.945Z ERROR   HLFConnectionManager    :fabric-client()          [client-utils.js]: sendPeersProposal - Promise is rejected: Error: chaincode error (status: 500, message: Error: The current identity must be activated (ACTIVATION_REQUIRED))
    at /home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/node_modules/grpc/src/node/src/client.js:554:15 {}$
2018-05-16T04:02:14.946Z ERROR   HLFConnection           :queryChainCode()         {"message":"Error trying to query business network. Error: chaincode error (status: 500, message: Error: The current identity must be activated (ACTIVATION_REQUIRED))","stack":"Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: The current identity must be activated (ACTIVATION_REQUIRED))\n    at channel.queryByChaincode.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:809:34)\n    at <anonymous>"}$
2018-05-16T04:02:14.946Z ERROR   HLFConnection           :ping()                   {"message":"Error trying to ping. Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: The current identity must be activated (ACTIVATION_REQUIRED))","stack":"Error: Error trying to ping. Error: Error trying to query business network. Error: chaincode error (status: 500, message: Error: The current identity must be activated (ACTIVATION_REQUIRED))\n    at _checkRuntimeVersions.then.catch (/home/damsen2/Chapter10/node_modules/composer-connector-hlfv1/lib/hlfconnection.js:713:34)\n    at <anonymous>"}$
2018-05-16T04:05:55.534Z ERROR   FileSystemCardStore     :get()                    {"message":"Unable to read card directory: /home/damsen2/.composer/cards/bob@pchardwareinc.com","stack":"Error: Unable to read card directory: /home/damsen2/.composer/cards/bob@pchardwareinc.com\n    at newErrorWithCause (/home/damsen2/Chapter10/node_modules/composer-common/lib/idcard.js:36:19)\n    at fs.stat.catch.cause (/home/damsen2/Chapter10/node_modules/composer-common/lib/idcard.js:323:19)\n    at <anonymous>","cause":{"errno":-2,"code":"ENOENT","syscall":"stat","path":"/home/damsen2/.composer/cards/bob@pchardwareinc.com"}}$
